harvard-lts/etd-integration-tests
# Etd 133

**ETD 133**
* * *

**JIRA Ticket**: https://jira.huit.harvard.edu/browse/ETD-133

IF PR:

# What does this Pull Request do?
this adds the etd-integration test

# How should this be tested?

test locally
1) cp example-env.txt .env
2) set .env and celeryconfig.py to dev values
3) bring up docker instance
` docker-compose -f docker-compose-local.yml up --build -d --force-recreate`
4)  https://localhost:10610/integration
you should see zero errors.

A description of what steps someone could take to:
* Reproduce the problem you are fixing (if applicable)
* Test that the Pull Request does what is intended.
* Please be as detailed as possible.
* Good testing instructions help get your PR completed faster.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n 
- integration tests? this is it

# Interested parties
Tag (@ mention) interested parties @awoods
